components/connectivity-adapter/internal/appregistry/service/converter.go

```diff
@@ -11,14 +12,23 @@ func NewConverter() *converter {
 	return &converter{}
 }
 
-func (c *converter) DetailsToGraphQL(in model.ServiceDetails) graphql.Application {
-	panic("not implemented")
+func (c *converter) DetailsToGraphQLInput(in model.ServiceDetails) (graphql.ApplicationRegisterInput, error) {
+	return graphql.ApplicationRegisterInput{
```

Comment:
Why you don't use `in` param?

Reply:
It would be nice to have test for this method.

Reply:
Added a comment to clarify what's going on here 🙂